.github/workflows/rtl-repo-sync.yml

```diff
@@ -3,6 +3,8 @@ name: RTL Repo Sync
 on:
   workflow_call:
   workflow_dispatch:
+permissions:
+  contents: write
```

Comment:
Does the PR creation logic work? I assume you need `pull-requests: write` here too, no?